python2verilog/frontend/generator2ir.py

```diff
@@ -37,9 +37,19 @@ def __init__(
         # pylint: disable=too-many-nested-blocks
         for node in context.py_ast.body:
             for assign in pyast.walk(node):
+                match assign:
+                    case pyast.Assign(
+                        targets=[pyast.Name(id=target_id)],
+                        value=pyast.Call(func=pyast.Name(id=func_id)),
+                    ):
+                        cxt = self._context.namespace[func_id]
+                        instance = cxt.create_instance(target_id)
+                        self._context.instances[target_id] = instance
+                continue
```

Comment:
What you think compared to code code? `continue` used to skip old code.

Reply:
Are you planning to have more "types" of `assign`?

If so, the `match` will work. Else, I think an `if` statement to avoid the `continue` is better

Reply:
> Are you planning to have more "types" of `assign`?

Well, I want to eventually support things like `a, b, c = geta(), something, input`.

> If so, the `match` will work. Else, I think an `if` statement to avoid the `continue` is better

The `continue` is there because I didn't want to comment out old code. Have you seen the old code yet? It should be immediately below the new code.

Reply:
The rewrite definitely looks cleaner compared to below. If you want to support multiple types then this code is a lot better.

You have to make sure it's functionally identical though.

Reply:
Given the tests pass, I think they're identical.